examples/opentelemetry-web/examples/fetchXhrB3/index.js

```diff
@@ -70,7 +70,7 @@ const prepareClickEvent = () => {
   const element1 = document.getElementById('button1');
   const element2 = document.getElementById('button2');
 
-  const clickHandler = (fetchFn) => {
+  const clickHandler = (fetchFn) => () => {
```

Comment:
double function intentional?

Reply:
Yes. Before, adding the click handler in L91/L92 invoked the function once on load, and the buttons did not do anything. They they were provided with `void` as a click handler. Now they are provided with a  `function(): void`.